Runtime/Model/Attributes/MachineAttributeProvider.cs

```diff
@@ -90,6 +90,10 @@ private void IncludeOsInformation(IDictionary<string, string> attributes)
             attributes["uname.version"] = UnityEngine.iOS.Device.systemVersion;
             attributes["uname.fullname"] = Environment.OSVersion.Version.ToString();
 #endif
+

```

Comment:
why we cannot use the if statement above ?

Reply:
UnityEngine.iOS.Device.systemVersion is iOS specific

Reply:
Yeah but in talking about environment.osversion